.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 .env.local
 public/sitemap.xml
 .vercel
+tsconfig.tsbuildinfo
```

Comment:
TypeScript 3.4 introduces a new flag called [incremental](https://www.typescriptlang.org/tsconfig#incremental) which tells TypeScript to save information about the project graph from the last compilation

It will use that information to detect the least costly way to type-check and emit changes to your project

In a nutshell, `tsconfig.buildinfo` is emitted to speed up the compilation!

Reference from the [docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html)
> By default with these settings, when we run tsc, TypeScript will look for a file called .tsbuildinfo in the output directory (./lib). If ./lib/.tsbuildinfo doesn’t exist, it’ll be generated. But if it does, tsc will try to use that file to incrementally type-check and update our output files.